File: frontendapp/lib/dtos/UserDTO.dart
class UserDTO{
  final String name;
  final String email;
  final String password;
  final String retypepassword;
  final String phonenumber;
  final String dateofbirth;
  String? address;

  UserDTO({
    required this.name,
    required this.email,
    required this.password,
    required this.retypepassword,
    required this.phonenumber,
    required this.dateofbirth,
    this.address,
  });

  Map<String,dynamic> toJson() {
    return {
      'name': name,
      'email': email,
      'password': password,
      'retypepassword': retypepassword,
      'phonenumber': phonenumber,
      'dateofbirth': dateofbirth,
      'address': address,
    };
  }
}

File: webadmin/src/app/dtos/user.dto.ts
export class UserDTO {
    id: number;
    name: string;
    email: string;
    password: string;
    phonenumber: string;
    address: string;
    dateofbirth: Date;
    createdat: Date;
    isactive: boolean;
    rolename: boolean;
    imagename: string;

    constructor(data: any) {
        this.id = data.id;
        this.name = data.name;
        this.email = data.email;
        this.password = data.password;
        this.phonenumber = data.phonenumber;
        this.address = data.address;
        this.dateofbirth = data.dateofbirth;
        this.createdat = data.createdat;
        this.isactive = data.isactive;
        this.rolename = data.rolename;
        this.imagename = data.imagename
    }
}